src/components/accessible-map.js

```diff
@@ -99,7 +99,7 @@ export class AccessibleMap extends LitElement {
 					(abbreviation) =>
 						svg`<g class="map__group">
 				<title id=${abbreviation}>${mapData[abbreviation].name}</title>
-				<a data-abbreviation=${abbreviation} @focus="${this._handlePointerEvent}"  href="https://tinykitelab.com" class="map__link map__link--${abbreviation}" aria-labelledby=${abbreviation}>
+				<a data-abbreviation=${abbreviation} @focus="${this._handlePointerEvent}"  href="/caselaw/?reporter=${abbreviation}" class="map__link map__link--${abbreviation}" aria-labelledby=${abbreviation}>
```

Comment:
@rebeccacremona -- in the original site, each state links to the reporters entries on the jurisdiction page for that site. This links to the first of the reporters in that jurisdiction. Is that what you want or do you want this to link to the jurisdiction (i.e. /caselaw/#illinois rather than /caselaw/?reporter=il)?

Reply:
Ooooh, thanks for that! Totally did not notice. Fixing!